packages/dicom/python/itkwasm-dicom-wasi/test/test_sort_dicom_series.py

```diff
@@ -0,0 +1,15 @@
+from pathlib import Path
+
+
+def test_sort_dicom_series():
+    from itkwasm_dicom_wasi import sort_dicom_series
+
+    file_name = "ImageOrientation.1.dcm"
+    test_file_path = Path(
+        "..", "..", "test", "data", "input", "DicomImageOrientationTest", file_name
+    )
```

Comment:
common.py for finding paths

Reply:
e.g. https://github.com/InsightSoftwareConsortium/itk-wasm/blob/main/packages/downsample/python/itkwasm-downsample-wasi/tests/common.py